plaidml/bridge/openvino/tests/functional/shared_tests_instances/single_layer_tests/detection_output.cpp

```diff
@@ -0,0 +1,91 @@
+// Copyright (C) 2020 Intel Corporation
+// SPDX-License-Identifier: Apache-2.0
+//
+
+#include "single_layer_tests/detection_output.hpp"
+
+using namespace LayerTestsDefinitions;
+
+namespace {
+
+const int numClasses = 5;
+const int backgroundLabelId = 0;
+const std::vector<int> topK = {3};
+// const std::vector<std::vector<int>> keepTopK = { {50}, {100} };
```

Comment:
You can comment 50 and 100 in next line like the following, usually we add a `,` after the last element in vector so clang-format will make each parameter one line (sometimes need to add `//` after each line).
```
const std::vector<std::vector<int>> keepTopK = {
// {50}, 
// {100}, 
{9},
{10}, 
{11},
};
```

Reply:
Oh right. I copied this from cpu plugin and it doesn't have the last comma. Will add it and check all other formats.